### PR TITLE
Update readme and release notes

### PR DESCRIPTION
**Pull request by jacekstan**
#### 1. Objective

Update general information about plugin in README.md and changes included into release in CHANGELOG.md for coming 2.3 Release.

#### 2. Description of change

CHANGELOG.md and README.md files.

#### 3. Quality assurance

Open README.md and CHANGELOG.md
and check formatting and correctness of information.

**🔧 Environments:**

- **Platform version**: Magento CE 2.2.4.
- **Omise plugin version**: Omise-Magento 2.2.
- **PHP version**: 7.0.29.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
 # CHANGELOG
 
+### [v2.3 _(Jun 20, 2018)_](https://github.com/omise/omise-magento/releases/tag/v2.3)
+
+#### ✨ Highlights
+
+- Introducing support of Alipay payments (PR [#113](https://github.com/omise/omise-magento/pull/113))
+- Support for Multistore Magento 2 configuration (PR [#117](https://github.com/omise/omise-magento/pull/117))
+- Simplify plugin installation process by using composer (PR [#112](https://github.com/omise/omise-magento/pull/112))
+- Introduce Webhook feature  (PR [#107](https://github.com/omise/omise-magento/pull/107))
```
Comment (guzzilar) [inline CHANGELOG.md]: Don't want to be too OCD but my eyes just spotted this 2 little spaces instead of 1 😆

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<p align="center"><a href='https://www.omise.co'><img src='https://assets.omise.co/assets/omise-logo-ed530feda8c7bf8b0c990d5e4cf8080a0f23d406fa4049a523ae715252d0dc54.svg' height='60'></a></p>
+<p align="center"><a href='https://www.omise.co'><img src='https://user-images.githubusercontent.com/10651523/41644923-b79aaf74-7499-11e8-9c7e-85186421c841.png' height='60'></a></p>
```
Comment (guzzilar) [inline README.md]: @jacekstan cool thanks for fixing this link!!

Btw, can you replace this url with https://cloud.githubusercontent.com/assets/2154669/26388730/437207e4-4080-11e7-9955-2cd36bb3120f.png instead? 🙇 🙇

File: CHANGELOG.md (hunk 1 above)
Comment (jacekstan) [reply CHANGELOG.md]: ok.. that has 0 influence on text visibility.. as it is formatted by wiki engine.. but of course.. will fix it..

Comment (guzzilar) [reply CHANGELOG.md]: lol I know I know 😂

**Pull request merged**